### PR TITLE
CORE-1982 -  Bar chart component

### PR DESCRIPTION
**Pull request by NathanAB**
changes: _rev_

### Proposed Changes
_Please use the Jira Key or NOREF followd by the changes_

- CORE-1982
  - Add bar chart component

### Checklist

- [ ] I have added or updated unit tests
- [ ] I have added or updated integration tests (if appropriate)
- [ ] I have added or updated documentation / readme (if appropriate)
- [ ] I have verified that my changes have not introduced new lint errors
- [ ] I have updated the change log

### Testing Instructions

image

### CODE REVIEW

**Review by suchak1 — APPROVED** [review]
lgtm

File: lib/components/BarChart/BarChart.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import { ResponsiveBar } from '@nivo/bar';
+import numeral from 'numeral';
+
+import { vds as designTokens } from '@/styles/build/js/design_tokens';
+import styles from './BarChart.css';
+
+const BAR_OFFSET = -45;
+const LABEL_OFFSET = 5;
+
+const CustomBarComponent = ({ x, y, width, height, label }) => (
```
Comment (kuguarpwnz) [inline lib/components/BarChart/BarChart.js:12]: let's move it to a separate file?

File: lib/components/BarChart/BarChart.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  return (
+    <div className={styles.barChartContainer}>
+      <ResponsiveBar
+        data={dataToShow}
+        margin={{ top: 50, right: 130, bottom: 50, left: 60 }}
+        axisLeft={{
+          legend: yLabel,
+          legendPosition: 'middle',
+          legendOffset: -20,
+          renderTick: () => <></>, // Do not render ticks
```
Comment (kuguarpwnz) [inline lib/components/BarChart/BarChart.js:42]:
```suggestion
          renderTick: () => null, // Do not render ticks
```

File: lib/components/BarChart/BarChart.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        }}
+        theme={{
+          textColor: designTokens.color.blue.darker.value,
+          fontFamily: designTokens.font.family.body.value,
+          fontSize: designTokens.font.size.sm.value,
+        }}
+        layout="horizontal"
+        indexBy="key"
+        borderRadius="5"
+        padding=".2"
```
Comment (kuguarpwnz) [inline lib/components/BarChart/BarChart.js:55]: I think we should use `={0.2}` for numeric values instead of passing a string

**Review by kuguarpwnz — APPROVED** [review]
👍

**Comment by virtru-github-public** [comment]
:rocket: PR was released in [`v1.15.0`](https://github.com/virtru/virtuoso-design-system/releases/tag/v1.15.0) :rocket:

**Pull request merged**